### PR TITLE
Don't send request where parameters are missing

### PR DESCRIPTION
**Pull request by nuruddeensalihu**
Hi,

Attached contained few changes that would restrict our api call when the params are missing.

### CODE REVIEW

File: src/LiveApi.js [hunk 1]
```diff
@@ -212,6 +212,9 @@ export default class LiveApi {
     }
 
     send(json) {
+        if (Object.keys(json).length === 0 && json.constructor === Object) {
```
Comment (qingweibinary) [inline src/LiveApi.js]: Did you test with undefined?

Object.keys(undefined) will throw error
I think 

`if (json && Object.keys(json).length === 0) ` would suffice

Comment (qingweibinary) [inline src/LiveApi.js]: Sorry, it should be `if (!json || Object.keys(json).length === 0)`

**Comment by borisyankov** [comment]
When do we need that?

**Comment by borisyankov** [comment]
This behavior is incorrect, even if we want it, and I am not saying we want it.
Do you expect a liveApi.send() to actually do nothing? I would expect it to fail, but do nothing?
This is not an improvement.

**Pull request closed without merging**